### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by Simon-Laux**

### CODE REVIEW

File: _locales/fr.xml [hunk 1]
```diff
@@ -1101,4 +1100,4 @@
     <string name="update_1_42_common">Nouveautés :\n\n💌 Désormais, les discussions à deux garantissent le chiffrement de bout en bout avec les contacts connus.\n\n😌 Pour plus de simplicité, ces contacts et discussions sont marqués d\'un badge vert.\n\n👥 Lorsque cela est possible, les groupes créés sont automatiquement chiffrés de bout en bout.\n\n🔄 Vos listes Accepté/Bloqué, Archivé, Épinglé et Sourdine sont synchronisées sur chacun de vos appareils.</string>
     <!-- placeholder will be replaced by the URL https://get.delta.chat/#changelogs -->
     <string name="update_1_42_android">Encore plus d\'améliorations à suivre ! Appareil photo, messages vocaux, sauvegardes complètes, lecteur d\'écran, arrière-plan différencié pour chaque compte, option \"sélectionner tout\" dans la galerie, panorama de webxdc, etc. Sinon, retrouvez-les toutes sur : %1$s</string>
-</resources>
+    </resources>
```
Comment (missytake) [inline _locales/fr.xml:1103]: this is weird, but xml is not sensitive to line breaks and indentation, right?

File: _locales/vi.xml [hunk 2]
```diff
@@ -1076,4 +1075,4 @@
     <string name="update_1_42_common">Có gì mới:\n\n💌 cuộc trò chuyện một-một đảm bảo mã hóa hai đầu cho những người liên hệ được giới thiệu\n\n😌 để mọi người đơn giản, chúng tôi đánh dấu những người liên hệ và cuộc trò chuyện này bằng dấu kiểm màu xanh lá cây\n\n👥 nhóm được tạo tự động với mã hóa hai đầu được đảm bảo nếu có thể\n\n🔄 Chấp nhận/Chặn, Đã lưu trữ, Đã ghim, Tắt tiếng được đồng bộ hóa trên tất cả các thiết bị của bạn</string>
     <!-- placeholder will be replaced by the URL https://get.delta.chat/#changelogs -->
     <string name="update_1_42_android">Hãy chú ý đến những cải tiến khác về máy ảnh, tin nhắn thoại, sao lưu tất cả, trình đọc màn hình, hình nền cho mỗi tài khoản, chọn tất cả thư viện, phong cảnh webxdc … hoặc đọc chúng tại %1$s</string>
-</resources>
+    </resources>
```
Comment (missytake) [inline _locales/vi.xml:1078]: and here.

**Review by missytake — APPROVED** [review]

File: _locales/fr.xml (hunk 1 above)
Comment (Simon-Laux) [reply _locales/fr.xml:1103]: no idea, this is all auto generated. also as far as I know indentation and line breaks son't matter that much in xml and html, at least outside of text content

File: _locales/tr.xml [hunk 3]
```diff
@@ -1103,4 +1104,5 @@
...
     <!-- placeholder will be replaced by the URL https://get.delta.chat/#changelogs -->
     <string name="update_1_42_android">Kamera, sesli iletiler, tümünü yedekleme, ekran okuyucusu, hesap başına duvar kâğıtları, galeride tümünü seç, webxdc manzarasındaki daha fazla iyileştirmeye dikkat edin … ya da onları %1$s adresinde okuyun</string>
-</resources>
+    <string name="update_1_44_android">Ne yeni:\n\n❤️ Evet! Bu uzun zamandır bekleniyordu: Tepki vermek için bir iletiye uzun dokunun\n\n🔗 QR kodlarını tarayamıyorsanız, onları çağırma bağlantısı olarak paylaşın\n\n🔵 Artık tüm hesapların okunmayan iletileri sayılır ve başlıkta gösterilir\n\n… ve TONLARCA hata düzeltmesi … %1$s\n\nUygulamanın iyileştirilmesine yardımcı olun: Gerçek dünya kullanımını daha da iyileştirmek için \"Ayarlar / Gelişmiş / İstatistikleri gönder\"i kullanarak geliştiricilere yardımcı olun. Göndermeden önce verilerinizi inceleyebilirsiniz</string>
+    </resources>
```
[the remark this answers is not in the document]
Comment (Simon-Laux) [reply _locales/tr.xml:1108]: I guess that's intentional here

**Pull request merged**